tests/integration.rs

```diff
@@ -5961,6 +5965,7 @@ mod tests {
         }
 
         #[test]
+        #[cfg(not(feature = "mshv"))]
```

Comment:
This should work. But I'm fine with disabling it until we're sure about this from our pipeline run.

Reply:
Yep, it minimizes the patch surface as otherwise we have to add `#[ignore]` every time some failing test is added.

Thanks